webserver/src/main/java/io/kestra/webserver/rooting/TenantAliasingRooter.java

```diff
@@ -0,0 +1,35 @@
+package io.kestra.webserver.rooting;
+
+import io.micronaut.context.annotation.Replaces;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.web.router.DefaultRouter;
+import io.micronaut.web.router.RouteBuilder;
+import io.micronaut.web.router.UriRouteMatch;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import java.net.URI;
+import java.util.Collection;
+import lombok.SneakyThrows;
+
+@Singleton
+@Requires(missingClasses = "io.kestra.ee.webserver.rooting.DefaultTenantAliasingRooter")
```

Comment:
I'd rather not add that here, it shows that we have an override in EE which we try to never show (information disclosure vulnerability)